aspnetcore/host-and-deploy/webconfig-transformation.md

```diff
@@ -0,0 +1,163 @@
+---
+title: web.config transformation
+author: guardrex
+description: Learn how to transform the web.config file when publishing an app.
+monikerRange: '>= aspnetcore-2.2'
+ms.author: riande
+ms.custom: mvc
+ms.date: 02/07/2019
+uid: host-and-deploy/webconfig-transformation
+---
+# web.config transformation
+
+By [Vijay Ramakrishnan](https://github.com/vijayrkn) and [Luke Latham](https://github.com/guardrex)
+
+Transformations to the *web.config* file can be applied automatically when an app is published based on:
+
+* [Configuration](#configuration)
```

Comment:
Given use of "configuration" in our doc set, it would be helpful to explain somewhere that here it refers to "build configuration"